h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -67,13 +95,52 @@ function SidebarInjector(chromeTabs, dependencies) {
     }
   };
 
+  /**
+   * Returns the type of content in a DOM element or document
+   */
+  this.detectContentType = detectContentType;
```

Comment:
Putting it on `this`? It's a shame that `detectContentType` is now "defined" in two different places (with two different docstrings).